### PR TITLE
Adds Butcherable Ordeals

### PR DESCRIPTION
**Pull request by EternityCoffee**
Adds three different meats and gives the ability to butcher sweepers, the indigo midnight, amber dusk, and violet dawn.

Changelog
:cl:
add: Added three different meats that can be obtained by butchering certain ordeals
code: added butchering to sweepers, indigo midnight, amber dusk, and violet dawn.
/:cl:

### CODE REVIEW

**Comment by ilikesaying** [comment]
finally, ordealburger

File: code/modules/mob/living/simple_animal/hostile/ordeal/amber.dm [hunk 1]
```diff
@@ -68,6 +68,8 @@
 	melee_damage_upper = 115 // If you get hit by them it's a major skill issue
 	pixel_x = -16
 	base_pixel_x = -16
+	butcher_results = list(/obj/item/food/meat/slab/human/mutant/worm = 2, /obj/item/food/meat/slab/human/mutant/worm = 3)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/hostile/ordeal/amber.dm:71]:
```suggestion
	butcher_results = list(/obj/item/food/meat/slab/human/mutant/worm = 3)
```

File: code/modules/mob/living/simple_animal/hostile/ordeal/indigo.dm [hunk 2]
```diff
@@ -14,6 +14,8 @@
 	armortype = BLACK_DAMAGE
 	melee_damage_lower = 20
 	melee_damage_upper = 24
+	butcher_results = list(/obj/item/food/meat/slab/human/mutant/sweeper = 2, /obj/item/food/meat/slab/human/mutant/sweeper = 1)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/hostile/ordeal/indigo.dm:17]:
```suggestion
	butcher_results = list(/obj/item/food/meat/slab/human/mutant/sweeper = 2)
```

File: code/modules/mob/living/simple_animal/hostile/ordeal/indigo.dm [hunk 3]
```diff
@@ -66,6 +68,8 @@
 	melee_damage_lower = 60
 	melee_damage_upper = 60
 	ranged = TRUE
+	butcher_results = list(/obj/item/food/meat/slab/human/mutant/sweeper = 3, /obj/item/food/meat/slab/human/mutant/sweeper = 4)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/hostile/ordeal/indigo.dm:71]:
```suggestion
	butcher_results = list(/obj/item/food/meat/slab/human/mutant/sweeper = 4)
```

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
Change case of the Ordeal_butchery.dmi
And is there any reason for it being a new file instead of putting it into food.dmi?

**Pull request merged**